server/services/discord/actions/on_reaction_add.js

```diff
@@ -1,17 +1,29 @@
 'use strict'
 
 const client = require('../init').client
-const { AreaGlue } = require('../../glue/glue.js')
+const { getActionFromCode, AreaGlue, checkActionParameters } = require('../../glue/glue.js')
+
 
 /**
  * @brief Triggered when a user reacts to a message.
  * @param {*} reaction Reaction object
  * @param {*} user User that reacted to the message
  */
-client.on('messageReactionAdd', (reaction, user) => {
+client.on('messageReactionAdd', async (reaction, user) => {
```

Comment:
Utility of async ?

Reply:
A forgotten property used during the tests